imap/jmap_mail_submission.c

```diff
@@ -1121,6 +1121,18 @@ static int getsubmission(struct jmap_get *get,
             json_object_del(sub, "envelope");
         }
 
+        /* created */
+        if (jmap_wantprop(get->props, "created")) {
+            char datestr[RFC3339_DATETIME_MAX];
```

Comment:
All the other TIME_MAX macros define the maximum length excluding the nul character, I assume that's also the case for RFC3339.

Reply:
RFC3339_DATETIME_MAX is is used as above throught Cyrus.  Its defined as 21 and time_to_rfc3339() always writes a UTC time string of length 20

Reply:
OK, great. it's weird that it is the only time MACRO that includes the nul byte (or the comments for the other macros are wrong). but I always found the MAX length definitions without nul byte to be asking for trouble anyway.